src/pages/ImportData.js

```diff
@@ -124,7 +124,21 @@ const ImportDataOverview = ({ header, snapshots, isDataset, snapshotResponses, u
         ])
       : url && div({ style: { fontSize: 16 } }, ['From: ', new URL(url).hostname]),
     div({ style: { marginTop: '1rem' } }, [
-      `The ${isDataset ? 'dataset' : 'snapshot'}(s) you just chose to import to Terra will be made available to you `,
+      !!url && isProtected(url, format)
+        ? [
+            icon('warning-standard', { size: 15, style: { marginRight: '0.25rem' }, color: colors.warning() }),
+            ' The data you chose to import to Terra are identified as protected and require additional security settings. Please select a workspace that has an Authorization Domain and/or protected data setting.',
```

Comment:
Does the UI indicate which workspaces have these attributes? E.g., does it filter the list of destination workspaces so that only valid targets are shown?

Reply:
It will after https://broadworkbench.atlassian.net/browse/AJ-1106 is done!  We recognize there will be a (hopefully brief) period of bad UX in between releasing this fix and aj-1106, but it's important to get the compliance fix out sooner rather than later!